app/main/views/services.py

```diff
@@ -15,13 +15,26 @@
 
 from ..auth import role_required
 from ..helpers.diff_tools import get_diffs_from_service_data, get_revision_dates
+from ..helpers.service import approve_or_download_sentence_for_framework
 
 
 @main.route('', methods=['GET'])
 @login_required
 @role_required('admin', 'admin-ccs-category', 'admin-ccs-sourcing')
 def index():
-    return render_template("index.html")
+    try:
+        frameworks = data_api_client.find_frameworks()
+    except HTTPError:
+        flash({'api_no_frameworks': True}, 'error')
+    else:
+        frameworks = [fw for fw in frameworks['frameworks'] if fw['status'] in ('standstill', 'live')]
+        frameworks = sorted(frameworks, key=lambda x: x['id'], reverse=True)
+        frameworks = [{'body': approve_or_download_sentence_for_framework(fw),
+                       'link': url_for('.list_agreements', framework_slug=fw['slug'], status='signed'),
+                       'title': '{name} agreements'.format(name=fw['name'])} for fw in frameworks]
+
+    return render_template("index.html",
+                           frameworks=frameworks)
```

Comment:
This could be a bit more of a descriptive name - we generally have `frameworks` coming out of our ears, and these are not the usual `framework` objects that you'd generally expect.  So something like `frameworks_for_countersigning`.

Actually, I'd probably be tempted to pass the (sorted) frameworks list in to the template and do the `body`, `link`, `title` stuff (including the `if framework['frameworkAgreementVersion']` logic) in the index.html template itself.